folding/src/examples/example_decomposable_folding.rs

```diff
@@ -194,6 +194,13 @@ fn constraints() -> BTreeMap<DynamicSelector, Vec<FoldingCompatibleExpr<TestFold
 
     let add = FoldingCompatibleExpr::Add(a.clone(), b.clone());
     let add = FoldingCompatibleExpr::Sub(add.into(), c.clone());
+    let add = FoldingCompatibleExpr::Sub(
+        add.into(),
+        Box::new(FoldingCompatibleExpr::Atom(
+            FoldingCompatibleExprInner::Constant(ScalarField::<TestFoldingConfig>::one()),
+        )),
+    );
+    // a + b + c + 1 = 0
```

Comment:
not sure if relevant, but in the expression you are subtracting, while in the comment you are adding.
the expression you have is `a + b - c - 1`